e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-sort.spec.js

```diff
@@ -0,0 +1,125 @@
+import Chance from 'chance';
+
+const relative = require('../../../relativeURL');
+const MOCK_USERS = require('../../../../fixtures/users');
+const { dashboardDeals } = require('../../../pages');
+const {
+  BSS_DEAL_MIA,
+  GEF_DEAL_DRAFT,
+} = require('../fixtures');
+
+const { BANK1_MAKER1, ADMIN } = MOCK_USERS;
+const chance = new Chance();
+
+context('Dashboard Deals filters - filter by multiple fields with multiple values', () => {
+  const ALL_DEALS = [];
+  const asda = { ...GEF_DEAL_DRAFT };
```

Comment:
can we change the name for this one? I forgot to do it when I pushed my changes

Reply:
done